doc/source/whatsnew/v0.17.0.txt

```diff
@@ -177,7 +177,28 @@ Other enhancements
 - ``pandas.tseries.offsets`` larger than the ``Day`` offset can now be used with with ``Series`` for addition/subtraction (:issue:`10699`).  See the :ref:`Documentation <timeseries.offsetseries>` for more details.
 
 - ``.as_blocks`` will now take a ``copy`` optional argument to return a copy of the data, default is to copy (no change in behavior from prior versions), (:issue:`9607`)
+- ``Period``, ``PeriodIndex`` and ``period_range`` can now accept multiplied freq (:issue:`7811`)
 
+  Multiplied freq represents a span of corresponding length. Below example creates a period of 3 days. Addition and subtraction will shift the period by its span.
+
+  .. ipython:: python
+
+     p = pd.Period('2015-08-01', freq='3D')
+     p
+     p + 1
+     p - 2
+     p.to_timestamp()
+     p.to_timestamp(how='E')
+
+  You can use multiplied freq in ``PeriodIndex`` and ``period_range``.
+
+  .. ipython:: python
+
+     idx = pd.period_range('2015-08-01', periods=4, freq='2D')
+     idx
+     idx + 1
+
+- ``Period.freq`` and ``PeriodIndex.freq`` are now stored as ``DateOffset`` instance, not ``str`` (:issue:`7832`)
 - ``regex`` argument to ``DataFrame.filter`` now handles numeric column names instead of raising ``ValueError`` (:issue:`10384`).
 - ``pd.read_stata`` will now read Stata 118 type files. (:issue:`9882`)
```

Comment:
did you regenerate the legacy pickle tests pretty recently? (as I had updated maybe 2-3 weeks ago). also pls add in msgpack ones (again just regenerate, they already exist)